sources/naturalearth.json

```diff
@@ -3,9 +3,11 @@
 	"fullname": "Natural Earth",
 	"name": "naturalearth",
 	"prefix": "ne",
-	"key":"id",
+	"key": "id",
```

Comment:
Natural Earth doesn't have IDs, so? @thisisaaronland what's the background on this?